python/cudf/dataframe/column.py

```diff
@@ -588,3 +588,10 @@ def to_dlpack(self):
         """{docstring}"""
         import cudf.io.dlpack as dlpack
         return dlpack.to_dlpack(self)
+
+    @property
+    def _handle(self):
```

Comment:
Should this be called _pointer?  I don't know about python @property.

Reply:
Yup - renamed

Reply:
`@property` just means this method acts like an attribute. In this specific case, it is a read-only attribute.

ref: https://docs.python.org/3/howto/descriptor.html#properties